components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -173,11 +180,12 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
                     index={channelIndex}
                 >
                     {(provided, snapshot) => {
+                        const {onTransitionEnd: draggableTransitionEnd, ...draggableProps} = provided.draggableProps;
```

Comment:
This is a bit obscure way of setting the `draggableTransitionEnd` variable and at the same time extracting the draggableProps. IMO that are too many responsabilities for a single line of code.

Reply:
✅  Updated as per request/discussion